_posts/0000-01-02-Ajju-kld.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+Use the left arrow to go back!
```

Comment:
welcome to github it is great
good to everybody
how are you
what can i do 
android is best